lnpbp-0031.md

```diff
@@ -40,8 +42,140 @@ License: CC0-1.0
 
 ## Specification
```

Comment:
It's not quite clear what the difference is between RGB30 and RGB31, and why RGB31 is so much more verbose.

Reply:
I'm not sure if I understand your question, but IRGB31 is a Standard Contractum Library, used to provide basic types to other interfaces.

Reply:
Interesting, good to know. The name is a little confusing, I thought it was like RGB21 but with decentralized issuance.